tests/lean/1275.lean.expected.out

```diff
@@ -13,7 +14,8 @@ fun x => do
   let scp ← getCurrMacroScope
   let mainModule ← getMainModule
   pure
-      { raw :=
+      {
+        raw :=
```

Comment:
This used to be formatted differently because we unconditionally printed a hard space after the opening brace.  Hard spaces should only be used if we know that they are not followed by other whitespace (like sepByIndent after the align change!), to prevent double spaces.

Reply:
why is this introducing an extra newline though? Shouldn't the align be sufficient in this case, so that no space needs to be introduced after `{`?

Reply:
That's the thing: there's no `.align` here since the structure instance doesn't have newline separators!  What we have here is a `.line` and apparently the formatter decided to put the whole `.fill` containing the multi-line field on a new line.